Task_7/app/model/Helper.js

```diff
@@ -0,0 +1,166 @@
+class Helper {
+  static checkerArray(arr) {
+    return typeof arr !== 'object' && arr === null && Array.isArray(arr);
+  }
+
+  static isString(str) {
+    return typeof str === 'string';
+  }
+
+  static isEmptyStr(str) {
+    return str.trim().length;
+  }
+
+  static checkCommentsArray(arrCom) {
+    if (Array.isArray(arrCom) && !arrCom.length) {
+      return true;
+    }
+    if (Array.isArray(arrCom) && arrCom.length > 0) {
+      const initComLength = arrCom.length;
+      const validateCommnets = arrCom.filter((com) =>
+        Comment.validateComment(com)
+      );
+
+      return initComLength === validateCommnets.length;
+    }
+  }
+
+  static isEmptyArray(arr) {
+    return Array.isArray(arr) && !arr.length;
+  }
+
+  static generateId(arr) {
+    const taskIds = arr.map((task) => Number(task.id)).sort((a, b) => a - b);
+    return String(taskIds.at(-1) + 1);
+  }
+
+  static sortKeysInObj(obj) {
+    return Object.keys(obj).sort();
+  }
+
+  static validateCommentValue(key, value) {
+    return !validateObj[key](value);
+  }
+
+  static validateComment(comment) {
+    return (
+      Helper.validateCommentValue('_id', comment.id) ||
+      Helper.validateCommentValue('description', comment.text) ||
+      Helper.validateCommentValue('_createdAt', comment.createdAt) ||
+      Helper.validateCommentValue('assignee', comment.author)
+    );
+  }
+
+  static checkerArray(arr) {
+    return Boolean(Array.isArray(arr) && arr.length);
+  }
+
+  static isHtmlNode(node) {
+    return node instanceof HTMLElement;
+  }
+
+  static isPrivate(value) {
+    return value ? TASK_PRIVATE.private : TASK_PRIVATE.public;
+  }
+
+  static checkLatinLetters(str) {
+    try {
+      if (!Helper.isString(str)) {
+        throw new Error(ERRORS.onlyString);
+      }
+
+      const strResult = str
+        .split('')
+        .filter((letter) => LATIN_LETTERS.includes(letter.toLowerCase()))
+        .join('');
+
+      return strResult === str;
+    } catch (error) {
+      console.error(error);
+      return false;
+    }
+  }
+
+  static checkKirillLetters(str) {
```

Comment:
you cam merge with checkLatinLetters . Add param to checkLatinLetters(str, **letters**)  and in  filter use letters 
 .filter((letter) => **letters**.includes(letter.toLowerCase()))